pallets/gear/src/schedule.rs

```diff
@@ -1097,8 +1136,7 @@ impl<T: Config> Default for MemoryWeights<T> {
             upload_page_data: cost(W::<T>::db_write_per_kb)
                 .saturating_mul(KB_AMOUNT_IN_ONE_GEAR_PAGE)
                 .saturating_add(T::DbWeight::get().writes(1)),
-            // TODO: make benches to calculate static page cost and mem grow cost (issue #2226)
-            static_page: Weight::from_parts(100, 0),
+            // TODO: make benches to calculate mem grow cost (issue #2226)
```

Comment:
This is now fixed - please remove this and append this issue as resolved in this PR

Reply:
Done